torch/csrc/jit/passes/reconstruct_scopes.cpp

```diff
@@ -89,7 +133,13 @@ void ReconstructScopesPass::run() {
   constructFunctionToModuleMap(*root_module);
   constructRelativeNamesForModules(*root_module, prefix);
 
-  visitBlock(graph->block());
+  if (has_duplicated_function) {
+    std::cout << "WARNING: There are some duplicated module instances." << std::endl;
```

Comment:
This is not clear. Maybe say your module seems to have two instances of the same type. Current debugging does not support distinguishing the two instances of the same type yet. Thus debugging information maybe incomplete. Or something to the effect.

Reply:
Yes, I'll fix that. One small question: Since it's a warning, is it ok to directly print it directly to std::cout like this?

Reply:
Good question. I missed this. I think there is something like `TORCH_WARN` or torch_debug soemthing. I will have to check.